test/commands/new_test.rb

```diff
@@ -225,6 +343,48 @@ def container_options
           content.must_match %(adapter type: :file_system, uri: ENV['CHIRP_TWO_DATABASE_URL'])
         end
       end
+
+      describe 'adapter option' do
+        describe 'mysql' do
+          let(:opts) { container_options.merge(adapter: 'mysql') }
+          it 'generates adapter config for mysql' do
+            content = @root.join('lib/chirp.rb').read
+            content.must_match %(adapter type: :sql, uri: ENV['CHIRP_DATABASE_URL'])
+          end
+        end
+
+        describe 'mysql2' do
+          let(:opts) { container_options.merge(adapter: 'mysql2') }
+          it 'generates adapter config for mysql2' do
+            content = @root.join('lib/chirp.rb').read
+            content.must_match %(adapter type: :sql, uri: ENV['CHIRP_DATABASE_URL'])
+          end
+        end
+
+        describe 'postgresql' do
+          let(:opts) { container_options.merge(adapter: 'postgresql') }
+          it 'generates adapter config for postgresql' do
+            content = @root.join('lib/chirp.rb').read
+            content.must_match %(adapter type: :sql, uri: ENV['CHIRP_DATABASE_URL'])
+          end
+        end
+
+        describe 'sqlite3' do
+          let(:opts) { container_options.merge(adapter: 'sqlite3') }
+          it 'generates adapter config for sqlite3' do
+            content = @root.join('lib/chirp.rb').read
+            content.must_match %(adapter type: :sql, uri: ENV['CHIRP_DATABASE_URL'])
+          end
+        end
+
+        describe 'memory' do
+          let(:opts) { container_options.merge(adapter: 'memory') }
+          it 'generates adapter config for memory' do
+            content = @root.join('lib/chirp.rb').read
+            content.must_match %(adapter type: :memory, uri: ENV['CHIRP_DATABASE_URL'])
```

Comment:
@joneslee85 Sure. I did that yesterday but that kind of broke the tests. It asks me a bunch of times when running the tests if I want to overwrite the `.lotusrc` file :) I will need to fix that somehow before committing or it will hang the tests on Travis.